conman/routes/models.py

```diff
@@ -52,7 +52,7 @@ class Route(PolymorphicMPTTModel):
         null=True,
         related_name='children',
     )
-    slug = models.SlugField(max_length=255, default='', help_text='''
+    slug = models.SlugField(max_length=255, default='', blank=True, help_text='''
```

Comment:
I think I missed the migration for this, and it needs validation in the admin form.

The slug is required unless there's no parent (That makes it the root). There can only be one root.